.travis.yml

```diff
@@ -70,127 +60,122 @@ rvm:
 
 matrix:
   include:
-    - rvm: 2.5.3
-      env: "GEM=railties"
-      sudo: required
-      before_install:
-        - "rm ${BUNDLE_GEMFILE}.lock"
-        - "travis_retry gem update --system"
-        - "travis_retry gem install bundler"
-        - "sudo sed -i 's/port = 5433/port = 5432/' /etc/postgresql/10/main/postgresql.conf"
-        - "sudo service postgresql restart 10"
-    - rvm: 2.6.0
-      env: "GEM=railties"
-      sudo: required
-      before_install:
-        - "rm ${BUNDLE_GEMFILE}.lock"
-        - "travis_retry gem update --system"
-        - "travis_retry gem install bundler"
-        - "sudo sed -i 's/port = 5433/port = 5432/' /etc/postgresql/10/main/postgresql.conf"
-        - "sudo service postgresql restart 10"
-    - rvm: ruby-head
-      env: "GEM=railties"
-      sudo: required
-      before_install:
-        - "rm ${BUNDLE_GEMFILE}.lock"
-        - "travis_retry gem update --system"
-        - "travis_retry gem install bundler"
-        - "sudo sed -i 's/port = 5433/port = 5432/' /etc/postgresql/10/main/postgresql.conf"
-        - "sudo service postgresql restart 10"
     - rvm: 2.5.3
       env: "GEM=actionview:ujs"
     - rvm: 2.5.3
       sudo: required
       env: "GEM=activejob:integration"
+      addons:
+        apt:
+          packages:
+          - rabbitmq-server
       services:
         - memcached
         - redis-server
-        - rabbitmq
+        - rabbitmq-server
       before_install:
+        - "rm ${BUNDLE_GEMFILE}.lock"
+        - "travis_retry gem update --system"
+        - "travis_retry gem install bundler"
         - sudo sed -i -e '/local.*peer/s/postgres/all/' -e 's/peer\|md5/trust/g' /etc/postgresql/*/main/pg_hba.conf
-        - "sudo sed -i 's/port = 5433/port = 5432/' /etc/postgresql/10/main/postgresql.conf"
         - "sudo service postgresql restart 10"
         - "[ -f /tmp/beanstalkd-1.10/Makefile ] || (curl -L https://github.com/beanstalkd/beanstalkd/archive/v1.10.tar.gz | tar xz -C /tmp)"
         - "pushd /tmp/beanstalkd-1.10 && make && (./beanstalkd &); popd"
     - rvm: 2.6.0
       sudo: required
       env: "GEM=activejob:integration"
+      addons:
+        apt:
+          packages:
+          - rabbitmq-server
       services:
         - memcached
         - redis-server
-        - rabbitmq
+        - rabbitmq-server
       before_install:
+        - "rm ${BUNDLE_GEMFILE}.lock"
+        - "travis_retry gem update --system"
+        - "travis_retry gem install bundler"
         - sudo sed -i -e '/local.*peer/s/postgres/all/' -e 's/peer\|md5/trust/g' /etc/postgresql/*/main/pg_hba.conf
```

Comment:
Is this still need?

Reply:
Using `ruby-head` caused this error without these lines. Then added these 3 lines to 2.6.0 and 2.5.3 to make these configuration consistent.

```ruby
    $ bundle install --jobs 3 --retry 3
    Traceback (most recent call last):
            2: from /home/travis/.rvm/rubies/ruby-head/bin/bundle:30:in `<main>'
            1: from /home/travis/.rvm/rubies/ruby-head/lib/ruby/2.7.0/rubygems.rb:302:in `activate_bin_path'
    /home/travis/.rvm/rubies/ruby-head/lib/ruby/2.7.0/rubygems.rb:283:in `find_spec_for_exe': Could not find 'bundler' (1.17.2) required by your /home/travis/build/rails/rails/Gemfile.lock. (Gem::GemNotFoundException)
    To update to the latest version installed on your system, run `bundle update --bundler`.\nTo install the missing version, run `gem install bundler:1.17.2`\n
```

Let me remove these lines from 2.6.0 and 2.5.3.

Reply:
I'm wondering if these 3 lines should be removed from ruby-head. 
@y-yagi Would you let me know your thoughts?

Reply:
Sorry to be confused. I was concerned about whether the `sudo sed -i -e '/local.*peer/s/postgres/all/' -e 's/peer\|md5/trust/g' /etc/postgresql/*/main/pg_hba.conf` line is still needed. Is this related to that error?

Reply:
I see. Removed `sudo sed -i -e '/local.*peer/s/postgres/all/' -e 's/peer\|md5/trust/g' /etc/postgresql/*/main/pg_hba.conf` and all CIs are running fine.